puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/tfm-ror52-release.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: tfm-ror52-release
+    project-type: pipeline
+    properties:
+      - github:
+          url: https://github.com/theforeman/rails-packaging
+    triggers:
+        - github
+        - pollscm:
+            cron: "* * * * *"
```

Comment:
This just means it's polling the repo to see if there is a new commit that was merged every minute?

Reply:
ACK

Reply:
I thought as I understood things in the docs, this is required for a trigger on merge. I could be wrong though and remove it to try it out.